oozie-to-airflow/run-sys-test

```diff
@@ -361,6 +365,15 @@ function prepare_configuration {
     fi
 }
 
+function prepare_gcloud_configuration {
+    if [[ -z "${COMPOSER_DAG_BUCKET}" ]]; then
+        COMPOSER_DAG_BUCKET=$(gcloud beta composer environments describe "${COMPOSER_NAME}" --location "${COMPOSER_LOCATION}" --format='value(config.dagGcsPrefix)')
+        COMPOSER_DAG_BUCKET=${COMPOSER_DAG_BUCKET%/dags}
+        COMPOSER_DAG_BUCKET=${COMPOSER_DAG_BUCKET#gs://}
+    fi
+    COMPOSER_WEB_UI_URL=$(gcloud beta composer environments describe "${COMPOSER_NAME}" --location "${COMPOSER_LOCATION}" --format='value(config.airflowUri)')
+}
```

Comment:
Please add echos that show the environment variables. I think it is super-important to show those derived values for any kind of debugging or diagnostics.

Reply:
I updated a PR.